internal/server/data/credentialrequest_test.go

```diff
@@ -0,0 +1,267 @@
+package data
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/assert/opt"
+
+	"github.com/infrahq/infra/internal/server/models"
+	"github.com/infrahq/infra/uid"
+)
+
+func TestPgNotifyRequestWithTriggers(t *testing.T) {
```

Comment:
I think these test names (this one and the one below for response) should be something like `TestCreateDestinationCredential`.  When testnames don't reference the function they are testing it becomes really difficult to maintain the test suite, and hard to audit what coverage exists. It's true these are indirectly testing triggers and notify, but primarily what we care about from our application is that a create results in a notify, so I think it's really `TestCreateDestinationCredential`.